overlord/devicestate/devicestate.go

```diff
@@ -116,6 +116,23 @@ func findSerial(st *state.State, device *auth.DeviceState) (*asserts.Serial, err
 	return a.(*asserts.Serial), nil
 }
 
+// findRevisionForModel returns the model assertion revision if any in the
+// assertion database for the given model, otherwise it returns -1.
+func findRevisionForModel(st *state.State, mod *asserts.Model) (modRevision int, err error) {
```

Comment:
findLastKnownRevisionForModel?

Reply:
the assertion db doesn't really let you find older revisions (unless different formats are involved), so it's always the case that we find the last-known

Reply:
maybe findKnownRevisionOfModel